src/FunctionApps/python/tests/GenerateCSVs/test_vxu.py

```diff
@@ -0,0 +1,55 @@
+import json
+import pathlib
+from GenerateCSVs.vxu import vxu_to_csv
+import pytest
+
+
+@pytest.fixture()
+def bundle():
+    return json.load(open(pathlib.Path(__file__).parent / "assets" / "vxu.json"))
+
+
+def test_elr_to_csv(bundle):
+    generated_rows = vxu_to_csv(bundle)
+    num_rows = 0
+    for row in generated_rows:
```

Comment:
$0.02: I liked the thing you (I think?) did the other day where you converted into a set to do unordered list equality

Reply:
I simplified by simply comparing to the full expected output.  I *think* list and set equality would both run in O(n), and the expected output ordering should be deterministic so I left it as an ordered list, but I might be missing something.